backend.py

```diff
@@ -785,29 +787,28 @@ def selection_confirmed_number(self):
                 selection_confirmed_number += 1
         return selection_confirmed_number
 
-    def get_number_flags_from_map(self):
+    def get_flag_count(self):
```

Comment:
nedáme jen `flag_count`? když se máme co nejvíc zbavit "get"

Reply:
Jestli je to metoda, kterou jepotřeba zavolat, nechte `get_`.